helmclient.go

```diff
@@ -361,6 +362,21 @@ func (c *Client) EnsureTillerInstalledWithValues(ctx context.Context, values []s
 // values provided when the chart was installed. The releaseName is the name
 // of the Helm Release that is set when the Helm Chart is installed.
 func (c *Client) GetReleaseContent(ctx context.Context, releaseName string) (*ReleaseContent, error) {
+	eventName := "get_release_content"
+
+	t := prometheus.NewTimer(controllerHistogram.WithLabelValues(eventName))
+
+	releaseContent, err := c.getReleaseContent(ctx, releaseName)
+	if err != nil {
+		controllerErrorGauge.WithLabelValues(eventName).Inc()
```

Comment:
I think the error masking won't work here and then the stack traces will be wrong. 

After updating the gauge I think we should return the masked error like we usually do.

Reply:
Good catch, never thought about that.